react/features/analytics/handlers/AmplitudeHandler.ts

```diff
@@ -79,7 +79,14 @@ export default class AmplitudeHandler extends AbstractHandler {
      */
     setUserProperties(userProps: any) {
         if (this._enabled) {
-            amplitude.getInstance().setUserProperties(userProps);
+            const identify = new Identify();
+
+            // Set all properties
+            Object.entries(userProps).forEach(([ key, value ]) => {
+                identify.set(key, value as any);
```

Comment:
Is this the equivalent?

Reply:
The new dependency doesn't have a setUserProperties method. We need to use the identify method with an Identify object.

Reply:
Yes, I can see that. Is that equivalent to the previous API, in terms of functionality?

Reply:
👍